students/adam_wulw/lesson_01_python_basics/apple_sharing.py

```diff
@@ -0,0 +1,3 @@
+s = int(input('How meny students?\n'))
+a = int(input('How meny apples?\n'))
```

Comment:
Try to use different variable names. Nobody knows what 'a' variable mens. a -> apple_number etc